scripts/export_comments.py

```diff
@@ -47,9 +47,36 @@ def create_comments_excel_file(survey_id, period, submissions):
         ws.cell(row, 2, submission.data['collection']['period'])
         ws.cell(row, 3, boxes_selected)
         ws.cell(row, 4, comment)
+        if survey_id == '134':
+            if '300w' in submission.data['data']:
+                ws.cell(row, 5, submission.data['data']['300w'])
+            if '300f' in submission.data['data']:
+                ws.cell(row, 6, submission.data['data']['300f'])
+            if '300m' in submission.data['data']:
+                ws.cell(row, 7, submission.data['data']['300m'])
+            if '300w4' in submission.data['data']:
+                ws.cell(row, 8, submission.data['data']['300w4'])
+            if '300w5' in submission.data['data']:
+                ws.cell(row, 9, submission.data['data']['300w5'])
+            checkboxes = ['91w', '92w1', '92w2', '94w1', '94w2', '95w', '96w', '97w',
+                          '91f', '92f1', '92f2', '94f1', '94f2', '95f', '96f', '97f',
+                          '191m', '192m1', '192m2', '194m1', '194m2', '195m', '196m', '197m',
+                          '191w4', '192w41', '192w42', '194w41', '194w42', '195w4', '196w4', '197w4',
+                          '191w5', '192w51', '192w52', '194w51', '194w52', '195w5', '196w5', '197w5']
+            filled_checkboxes = ""
+            for checkbox in checkboxes:
+                if checkbox in submission.data['data']:
+                    filled_checkboxes = filled_checkboxes + f"{checkbox}, "
+            ws.cell(row, 10, filled_checkboxes)
```

Comment:
Why are you adding a new row at the end for the checkboxes when we already have a place for checkboxes to go (it's in the boxes_selected line above).
You could quickly sort it by changing this line to ` ws.cell(row, 3, filled_checkboxes)` but that's kinda messy, could you.  I think it should follow a common pattern where possible.